src/shared/istiopolicy/creator.go

```diff
@@ -246,3 +286,27 @@ func (c *Creator) generateAuthorizationPolicy(
 
 	return newPolicy
 }
+
+func (c *Creator) intentsHTTPResourceToIstioOperations(resources []v1alpha2.HTTPResource) []*v1beta12.Operation {
+	operations := make([]*v1beta12.Operation, 0, len(resources))
+
+	for _, resource := range resources {
+		operations = append(operations, &v1beta12.Operation{
+			Methods: c.intentsMethodsToIstioMethods(resource.Methods),
+			Paths:   []string{resource.Path},
+		})
+	}
+
+	return operations
+}
+
+func (c *Creator) intentsMethodsToIstioMethods(intent []v1alpha2.HTTPMethod) []string {
+	istioMethods := make([]string, 0, len(intent))
+	for _, method := range intent {
+		// Istio documentation specifies "A list of methods as specified in the HTTP request" in uppercase
+		strMethod := strings.ToUpper(string(method))
```

Comment:
You can change the definition of HTTPMethod so that instead of having hardcoded constants, it refers to http.MethodGet, etc.
But right now I see it's already uppercase, so you don't need to worry about it - it will be uppercase anyway